app/controllers/v1/songs_controller.rb

```diff
@@ -1,11 +1,11 @@
 class V1::SongsController < ApplicationController
   def index
     songs = Song.all
-    render json: songs
+    render json: songs, include: [:artist]
```

Comment:
:+1: 

You probably also want to eager load artists here to avoid N+1's: `Song.includes(:artist)`